src/csharp/generate_proto_csharp.sh

```diff
@@ -24,12 +24,12 @@ HEALTHCHECK_DIR=src/csharp/Grpc.HealthCheck
 REFLECTION_DIR=src/csharp/Grpc.Reflection
 TESTING_DIR=src/csharp/Grpc.IntegrationTesting
 
-$PROTOC --plugin=$PLUGIN --csharp_out=$EXAMPLES_DIR --grpc_out=$EXAMPLES_DIR \
+$PROTOC --plugin=$PLUGIN --csharp_out=$EXAMPLES_DIR --grpc_out=$EXAMPLES_DIR --grpc_opt=use_buffer_serialization \
```

Comment:
btw, you'll also need to set the "--csharp_opt=use_buffer_serialization" so that the messages also get generated with the right methods (which can confuse users IMHO).

Reply:
Yes, but I didn't want to do this yet since I had to build a local version of protoc that can handle that option. Given that the plugin and protoc are separate, I opted to only test out the grpc_opt in this PR for now.